addons/udes_stock/controllers/stock_move_line.py

```diff
@@ -37,14 +37,14 @@ def suggested_locations(self, move_line_ids):
 
         if locations:
             response.append({"title": "",
-                             "locations": locations.get_info()})
+                             "locations": locations.sorted(lambda l: l.name).get_info()})
```

Comment:
For consistency, can we sort them in the function just before returning them?  This way in case it is used other places we will use them in the same order, example https://github.com/unipartdigital/udes-open/blob/master/addons/udes_stock/models/stock_picking.py#L1925